onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -0,0 +1,563 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Any reason this file is part of session, and not framework?

Reply:
No reason, should I move it into framework?

Reply:
It's including headers from core/session though, so it shouldn't be below that should it? Otherwise it creates a circular dependency. 

---
In reply to: [419748718](https://github.com/microsoft/onnxruntime/pull/3314#discussion_r419748718) [](ancestors = 419748718)

Reply:
I moved it into framework